From dc796bdb3c45c99f34ae772255623462b5bcec84 Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Tue, 18 Aug 2020 13:55:59 +0900
Subject: [PATCH 4/8] lang: Add autotools (ac, am, in, m4, pc)

---
 src/lang.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/lang.c b/src/lang.c
index c74af19..d90ddb9 100644
--- a/src/lang.c
+++ b/src/lang.c
@@ -12,6 +12,7 @@ lang_spec_t langs[] = {
     { "asm", { "asm", "s" } },
     { "asp", { "asp", "asa", "aspx", "asax", "ashx", "ascx", "asmx" } },
     { "aspx", { "asp", "asa", "aspx", "asax", "ashx", "ascx", "asmx" } },
+    { "autotools", { "ac", "am", "in", "m4", "pc" } },
     { "batch", { "bat", "cmd" } },
     { "bazel", { "bazel" } },
     { "bitbake", { "bb", "bbappend", "bbclass", "inc" } },
-- 
2.27.0

